jettons/PEPE.yaml

```diff
@@ -0,0 +1,11 @@
+name: PEPE`S TON CLUB
+description: "Pepe’s Ton Club: Where All Frogs Get Rich and Have Fun!"
+image: "https://cache.tonapi.io/imgproxy/J4eaPMU00YJrGAAFC0Vu8w0GEOfVWkL4-HtTh5cAcis/rs:fill:200:200:1/g:no/aHR0cHM6Ly9yb3NlLW9iZWRpZW50LW1pdGUtNTA5Lm15cGluYXRhLmNsb3VkL2lwZnMvUW1ZOGF3TFg1ellNU2RkNTh1TlFjb0wzb0FpTURGNDI0NTNHcGhBSnUxQUxtQw.webp"
```

Comment:
Please don't use this link, it's internal cache of tonapi.io. 
Your image has to be available at any other hosting.

Reply:
Done, changed.